cmd/kubelet/app/server.go

```diff
@@ -39,9 +39,9 @@ import (
 	"k8s.io/mount-utils"
 
 	cadvisorapi "github.com/google/cadvisor/info/v1"
-	"go.opentelemetry.io/otel/exporters/otlp/otlpgrpc"
+	"go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc"
 	otelsdkresource "go.opentelemetry.io/otel/sdk/resource"
-	"go.opentelemetry.io/otel/semconv"
+	"go.opentelemetry.io/otel/semconv/v1.10.0"
```

Comment:
`v1.12.0` is the latest sem version. I wonder why not use it? https://pkg.go.dev/go.opentelemetry.io/otel/semconv

Reply:
Done!